docs/provider/static.rst

```diff
@@ -30,33 +31,57 @@ already running resources.
                   # during the deployment (e.g HAProxy)
                   # At least 5 IPs routable through the
                   # APIs interface
+                  - 198.51.100.49
+                  - 198.51.100.50
+                  - 198.51.100.51
+                  - 198.51.100.52
+                  - 198.51.100.53
        # Openstack external network will be configured according
-       # to the following keys/values. Ips have to be routable through
+       # to the following keys/values. IPs have to be routable through
        # the external interface
-       start:   # First available ip
-       end:     # Last available ip
-       cidr:    # cidr notation to describe the pool of ips
-       gateway: # Ip address of the gateway
-       dns:     # Ip address of the DNS
+       start:   203.0.113.17     # First available IP
+       end:     203.0.113.31     # Last available IP
+       cidr:    203.0.113.16/28  # cidr notation to describe the pool of IPs
+       gateway: 203.0.113.254    # IP address of the gateway
+       dns:     203.0.113.253    # IP address of the DNS
      eths:
        - eth1   # Name of the APIs interface
        - eth2   # Name of the external interface
-
    resources: # An object with roles and there associated resources
      control:   # The role control, followed by resource info:
-       address:   # Ip of the control node
-       alias:     # Name for the node (optional)
-       user:      # ssh user name (optional)
-       keyfile:   # path to ssh private key (optional)
-       port:      # ssh port (optional)
-       extra:     # Extra variables passed to ansible (optional)
-         ansible_become: yes
-
+       address:   192.0.2.17  # IP of the control node
+       alias:     control   # Name for the node (optional)
+       user:      user      # ssh user name (optional)
+       keyfile:   keyfile   # path to ssh private key (optional)
+       port:      22        # ssh port (optional)
+       extra:   # Extra variables passed to ansible (optional)
+         ansible_become: yes  # currently mandatory for a successful deploy
      compute:   # A role can also have a list of resources
-       - address: # Ip address of the first compute
-         alias:   # ...
-         user:    # ...
-         keyfile: # ...
-         port:    # ...
-       - address: # Ip address of the second compute
-       - address: # Ip address of the third compute
+       - address: 192.0.2.33  # IP address of the first compute
+         alias:   compute1  # ...
+         user:    user      # ...
+         keyfile: keyfile   # ...
+         port:    22        # ...
+         extra:   # Extra variables passed to ansible (optional)
+           ansible_become: yes  # currently mandatory for a successful deploy
```

Comment:
Actually it depends on your setup. It's required  if the `user` needs `sudo` to do priviledged things.

Reply:
Ok, sorry, I though that nowadays is almost everywhere force to connect as unprivileged user... I Will rewrite those lines (tomorrow, I guess)